dashboard/estar.html

```diff
@@ -148,12 +148,15 @@
 		          	<div class="row border-bottom">
 		          		<div class="col-xs-3">
 		          			<div class="row center">
-		          				<span class="circle-chart" id="energy-star-score-radial"></span>
+		          				<!-- <span class="circle-chart" id="energy-star-score-radial"></span> -->
+		          				<div class="local-ranking-container"><span class="local-ranking-lg">#<span class="building-ranking" class="local-ranking"></span></span><br>
+		          				out of 100<span class="center"><a data-toggle="tooltip" data-placement="right" title="The local ranking figure compares peer buildings by their ENERGY STAR Score. In cases where the rank is identical, a building's site EUI is also used. The higher a ranking, the better a building's performance.">  <i class="fa fa-info-circle" aria-hidden="true"></i></a></span></div>
```

Comment:
The 100 in  "X out of 100" is variable now that this isn't an energy star score; but instead local ranking.  There's a change in #86 that takes care of this, but depending on which branch gets merged first, we should make sure the final product is something like  `out of <span class="local-ranking-length">100</span>`